Makefile

```diff
@@ -199,4 +200,12 @@ helm-package: generate manifests
 
 .PHONY: frigate
 frigate:
-	frigate gen chart/validator-plugin-azure --no-deps -o markdown > chart/validator-plugin-azure/README.md
\ No newline at end of file
+	frigate gen chart/validator-plugin-azure --no-deps -o markdown > chart/validator-plugin-azure/README.md
+
+# dev has a cleanup step right now which deletes the old Docker image. Without
```

Comment:
You can safely remove this. DevSpace reusing the dev docker image is not an issue. The dev image basically just contains delve and Go. The Go source is synchronized from your local, so you don't need to update this image.

Reply:
Thanks. Removed this step but kept the target for an easy way to start DevSpace.